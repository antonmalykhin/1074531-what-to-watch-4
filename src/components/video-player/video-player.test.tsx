import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const filmPreviewSettings = {
  WIDTH: 280,
  HEIGHT: 175,
  LOOP: true,
  SRC: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  POSTER: `img/the-grand-budapest-hotel-poster.jpg`,
  IS_PLAYING: false
};

it(`Should VideoPlayer render correctly`, () => {
  const tree = renderer
    .create(
        <VideoPlayer
          width={filmPreviewSettings.WIDTH}
          height={filmPreviewSettings.HEIGHT}
          src={filmPreviewSettings.SRC}
          poster={filmPreviewSettings.POSTER}
          loop={filmPreviewSettings.LOOP}
          isPlaying={filmPreviewSettings.IS_PLAYING}
        />, {
          createNodeMock: () => {
            return {};
          }
        }).toJSON();

  expect(tree).toMatchSnapshot();
});
