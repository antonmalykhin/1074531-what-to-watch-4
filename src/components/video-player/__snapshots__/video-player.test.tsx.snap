// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  height="175"
  poster="img/the-grand-budapest-hotel-poster.jpg"
  width="280"
/>
`;
